tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -4046,8 +4047,193 @@ GetCudnnFusedMatmulGraph(dnn::DataType input_type, dnn::DataType bias_type,
   return std::make_unique<cudnn_frontend::OperationGraph>(std::move(op_graph));
 }
 
+tsl::StatusOr<std::unique_ptr<cudnn_frontend::OperationGraph>>
+GetCudnnFusedMHASimpleOperationGraph(
```

Comment:
In terms of slicing PRs, MHA support in SE could probably land by itself (is there a way to e2e test it just in SE?)

Reply:
Unfortunately no. There is no way that I can think of that we test SE changes only. Do u have any suggestions?

Reply:
OK indeed we don't really test things on SE level (probably an oversight), but then I think it's lesser evil to commit SE stuff first (even without tests =/)

Reply:
In that case, we can create a PR with SE changes for all the patterns that we intend to have?

Reply:
@cheshire Any comments on the above?

Reply:
Yes please.